man-roxygen/args-methods-reserved.R

```diff
@@ -0,0 +1,2 @@
+#' @param reserved Logical. Indicates whether reserved variables should be
+#' included in the output. Defaults to \code{FALSE}.
```

Comment:
Should we mention what the reserved variables are? I guess we'd have to remember to edit this when adding additional reserved variables, but it could be good to document them here. I'm OK either way.

Reply:
Added `?'reserved-variables` doc page.